util/check_tool_requirements.py

```diff
@@ -6,15 +6,39 @@
 from distutils.version import StrictVersion
 import logging as log
 import os
+import re
 import subprocess
 import sys
 
 # Display INFO log messages and up.
 log.basicConfig(level=log.INFO, format="%(levelname)s: %(message)s")
 
-# Populate __TOOL_REQUIREMENTS__
-topsrcdir = os.path.join(os.path.dirname(__file__), '..')
```

Comment:
Please keep the root of the repository as a separate variable somewhere to better understand the hierarchy and make it easier to adjust as the file is moved around.

Reply:
Done.